config.json

```diff
@@ -229,23 +229,23 @@
     "optimizedCheckout-logo-size": "250x100",
     "optimizedCheckout-logo-position": "left",
     "optimizedCheckout-headingPrimary-color": "#333333",
-    "optimizedCheckout-headingPrimary-font": "Google_Montserrat_400",
+    "optimizedCheckout-headingPrimary-font": "Google_Montserrat_600",
     "optimizedCheckout-headingSecondary-color": "#333333",
-    "optimizedCheckout-headingSecondary-font": "Google_Montserrat_400",
+    "optimizedCheckout-headingSecondary-font": "Google_Montserrat_600",
     "optimizedCheckout-body-backgroundColor": "#ffffff",
     "optimizedCheckout-colorFocus": "#4496f6",
     "optimizedCheckout-contentPrimary-color": "#333333",
-    "optimizedCheckout-contentPrimary-font": "Google_Karla_400",
+    "optimizedCheckout-contentPrimary-font": "Google_Montserrat_500",
     "optimizedCheckout-contentSecondary-color": "#757575",
-    "optimizedCheckout-contentSecondary-font": "Google_Karla_400",
-    "optimizedCheckout-buttonPrimary-font": "Google_Karla_400",
+    "optimizedCheckout-contentSecondary-font": "Google_Montserrat_500",
+    "optimizedCheckout-buttonPrimary-font": "Google_Montserrat_500",
     "optimizedCheckout-buttonPrimary-color": "#ffffff",
     "optimizedCheckout-buttonPrimary-colorHover": "#ffffff",
     "optimizedCheckout-buttonPrimary-colorActive": "#ffffff",
     "optimizedCheckout-buttonPrimary-backgroundColor": "#333333",
     "optimizedCheckout-buttonPrimary-backgroundColorHover": "#666666",
     "optimizedCheckout-buttonPrimary-backgroundColorActive": "#000000",
-    "optimizedCheckout-buttonPrimary-borderColor": "#cccccc",
+    "optimizedCheckout-buttonPrimary-borderColor": "transparent",
```

Comment:
Is `transparent` a valid value for a color picker ?

Reply:
🤷‍♀️ not sure but I guess I could just pick the backgroundColor then so it looks as transparent

Reply:
Can you bundle the theme and test the change inside theme editor

Reply:
It definitely comes up in the editor, but once you change it to something else, you can make it transparent.

<img width="750" alt="Screen Shot 2019-04-11 at 5 29 04 pm" src="https://user-images.githubusercontent.com/1621894/55938783-7da3ce80-5c7f-11e9-97e2-3b5b6dec57a8.png">